android/src/main/java/com/swmansion/reanimated/NodesManager.java

```diff
@@ -53,6 +53,16 @@ public void scrollTo(int viewTag, double x, double y, boolean animated) {
     NativeMethodsHelper.scrollTo(view, x, y, animated);
   }
 
+  public void dispatchCommand(int viewTag, String commandId, ReadableArray commandArgs) {
+    mContext.runOnNativeModulesQueueThread(
+        new GuardedRunnable(mContext.getExceptionHandler()) {
+          @Override
+          public void runGuarded() {
+            mUIManager.dispatchCommand(viewTag, commandId, commandArgs);
```

Comment:
Technically, this will be run on the `mqt_native_module` thread so there may be a slight delay if we don't synchronize it properly with the UI thread. Instead, maybe we should push commands to the queue here and flush the queue in `performOperations` which is already synchronized with the UI thread?